lib/shopify-cli/commands/deploy/heroku.rb

```diff
@@ -143,30 +141,30 @@ def heroku_command
 
         def heroku_create_new_app
           output, status = @ctx.capture2e(heroku_command, 'create')
-          raise(ShopifyCli::Abort, '{{x}} Heroku app could not be created') unless status.success?
+          @ctx.error('Heroku app could not be created') unless status.success?
           @ctx.puts(output)
 
           new_remote = output.split("\n").last.split("|").last.strip
           result = @ctx.system('git', 'remote', 'add', 'heroku', new_remote)
-          msg = "{{x}} Heroku app created, but couldn’t be set as a git remote"
-          raise(ShopifyCli::Abort, msg) unless result.success?
+          msg = "Heroku app created, but couldn’t be set as a git remote"
+          @ctx.error(msg) unless result.success?
```

Comment:
What was previously printed in `{{x}}`? Will that still be printed in the new `@ctx.error` flow?

Reply:
`@ctx.error() ` prefixes `{{x}} ` to the message parameter passed to it, hence it's removed in any message that is now passed to `#error()`